src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/transpose.cpp

```diff
@@ -4,57 +4,49 @@
 
 #include <vector>
 
-#include "single_layer_tests/transpose.hpp"
+#include "single_op_tests/transpose.hpp"
```

Comment:
apply clang-format

Reply:
Let's keep it as is. Clang-format should be enabled by plugin owner